src/components/Card.js

```diff
@@ -30,7 +30,7 @@ export const CardList = (props) => {
   if (className === 'home-articles-cont') {
     button = (
       <div className="read-more-btn">
-        <NavLink to="/login" className="category-link">
+        <NavLink to="/articles/category/sports" className="category-link">
```

Comment:
make this line to={`/articles/category/${title}`} so that the button can link to the correct category page for all article containers.